src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,55 @@
+package duke.task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Class that represents an event item in the taskList.
+ */
+public class Event extends Task{
```

Comment:
Perhaps you could add an extra space between `Task` and `{` to follow the coding standards and practices?